BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
+
+// REST API to get exchange rates based on input currencies
+app.MapPost("/", service.GetExchangeRates);
```

Comment:
We are retrieving data with this endpoint. So, by convention we should use a GET method instead of a POST one.

```suggestion
app.MapGet("/", service.GetExchangeRates);
```